spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -802,3 +802,113 @@ definitions:
+
+ - OrbitClockBound:
+     desc: Orbit and clock bound.
+     fields:
+       - sat_id:
+           type: u8
+           desc: >
+             Satellite ID. Encoded following either RTCM DF068 (GPS), DF252 (Galileo),
+             or DF488 (BDS) according to the constellation.
+       - orb_radial_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_radial_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Mean (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Standard Deviation (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+
+ - MSG_SSR_ORBIT_CLOCK_BOUNDS:
+     id: 0x05DE
+     short_desc: Combined Orbit and Clock Bound
+     public: False
+     fields:
+       - time:
+           type: GPSTimeSec
+           desc: GNSS reference time of the bound
+       - nb_msg_dataset:
+           type: u8
+           desc: Number of messages in the dataset
+       - id_msg_dataset:
+           type: u8
+           desc: Position of this message in the dataset
+       - ssr_update_interval:
+           type: u8
+           desc: >
+             Update interval between consecutive bounds. Encoded following
+             RTCM DF391 specification.
```

Comment:
I think these can be placed in a header type (see for example how it works for the STECHeader here: https://github.com/swift-nav/libsbp/blob/2c9688ad3ae3effc4714cc7e50c2ea994c9546e8/spec/yaml/swiftnav/sbp/ssr.yaml#L375).
The variables here will be similar in the other upcoming messages (SSR_CODE_PHASE_BIASES_BOUNDS, new SSR_STEC_CORRECTION, SSR_GRIDDED_CORRECTION_BOUNDS).
In the header, let's name `nb_msg_dataset` and `id_msg_dataset` as `num_msgs` and `seq_num` instead, then we follow the existing practice already in libsbp.
These are the fields I'd suggest for the header type:
time (GPSTimeSec), num_msgs, seq_num, update_interval, solution_id.
Note that iod is not in the above list because it's naming changes slightly for the gridded and stec corrections.

Reply:
Does the order matter? For example in the definition of MSG_SSR_ORBIT_CLOCK_BOUNDS, IOD comes before solution ID but in MSG_SSR_STEC_CORRECTION it comes after. So the header type isn't exactly the same for both.

Reply:
Good call, we should update the Excel sheet so the solution ID comes before in both. And then solution ID can be in the common header here.